addons/l10n_ph/models/account_tax.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    l10n_ph_atc = fields.Char("Philippines ATC")
```

Comment:
what's ATC? can you add a help tooltip ?


I don't see any use of this field anywhere, btw.

Reply:
Its for the BIR report. Its init from the Template so we can use it in the BIR report in later module